From 94b3fa50da5b15d206c13ef9d9a591862407ace8 Mon Sep 17 00:00:00 2001
From: Yves Vandervennet <yvanderv@opensource.altera.com>
Date: Wed, 28 Oct 2015 02:41:04 -0500
Subject: [PATCH 010/151] FogBugz #332657-2: add Stratix10 SWVP to arm64
 defconfig

commit  0ef782e0af08d90b5a2c5c666f6f58bc44ec42ae from
https://github.com/altera-opensource/linux-socfpga.git
branch is socfpga-5.4.64-lts

Changes to the ARM64 defconfig to enable the Stratix10 SWVP

Signed-off-by: Yves Vandervennet <yvanderv@opensource.altera.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
Integrated-by: Jun Zhang <jun.zhang@windriver.com>
---
 arch/arm64/boot/dts/altera/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/altera/Makefile b/arch/arm64/boot/dts/altera/Makefile
index 27bb925adc8d..63ee9369b884 100644
--- a/arch/arm64/boot/dts/altera/Makefile
+++ b/arch/arm64/boot/dts/altera/Makefile
@@ -1,2 +1,3 @@
 # SPDX-License-Identifier: GPL-2.0-only
 dtb-$(CONFIG_ARCH_STRATIX10) += socfpga_stratix10_socdk.dtb
+dtb-$(CONFIG_ARCH_STRATIX10SWVP) += stratix10_swvp.dtb
-- 
2.26.1

